website/pages/docs/advanced-topics/monitoring/otel-grafana.md

```diff
@@ -0,0 +1,253 @@
+---
+title: OpenTelemetry and Grafana
+description: Walkthrough on how to use OpenTelemetry to monitor CloudQuery Syncs with Grafana
+---
+
+# OpenTelemetry and Grafana
+
+In this example we will show how to visualize [CloudQuery][CloudQuery] [OpenTelemetry][OpenTelemetry] traces, metrics and logs with [Grafana][Grafana].
+We will use [Docker Compose][Compose] to run [Grafana][Grafana] and related services, so make sure you it installed on your machine.
+
+## Step 1: Creating a Docker Compose file
+
+We will use [Tempo][Tempo] for ingesting traces, [Loki][Loki] for logs, [Prometheus][Prometheus] for metrics, and the [OpenTelemetry collector][Collector] for collecting and forwarding the data to each service.
+
+Create a file named `docker-compose.yml` with the following content:
+
+```yaml
+version: "3.8"
+services:
+  tempo:
+    image: grafana/tempo:latest
+    command: ["-config.file=/etc/tempo.yaml"]
+    volumes:
+      - tempo_data:/tmp
+      - ./tempo/tempo.yaml:/etc/tempo.yaml
+    ports:
+      - "3200"
+      - "4318"
+  loki:
+    image: grafana/loki:latest
+    ports:
+      - "3100:3100"
```

Comment:
Is there a meaning to sometimes using the explicit mapping of identical ports, and sometimes doing it implicitly?

Reply:
So the mapping of identical ports so they can be used from the host. We only need to expose Grafana's `3000` and the collector's `4318` to the host so I'll clean this up.